packages/terra-site/themeable-variables.json

```diff
@@ -1 +1,278 @@
+    "--terra-menu-submenu-header-title-margin": "7px 0 7px 0",
+    "--terra-menu-submenu-header-buttons-icon-color": "#1c1f21",
+    "--terra-menu-submenu-header-buttons-icon-font-size": "12px",
+    "--terra-menu-submenu-header-buttons-margin": "0",
+    "--terra-menu-submenu-header-buttons-padding": "8px",
+    "--terra-menu-submenu-header-buttons-hover-color": "#111",
+    "--terra-menu-fullscreen-header-background-color": "#fff",
+    "--terra-menu-fullscreen-header-border-bottom": "1px solid #dedfe0",
+    "--terra-menu-fullscreen-header-title-color": "#1c1f21",
+    "--terra-menu-fullscreen-header-title-font-size": "1rem",
+    "--terra-menu-fullscreen-header-title-line-height": "1.2",
+    "--terra-menu-fullscreen-header-title-margin": "0.5714rem 0 0.5714rem 0.3571rem",
+    "--terra-menu-fullscreen-header-buttons-icon-font-size": "1.45rem",
+    "--terra-menu-fullscreen-header-buttons-margin": "0.3571rem 0",
+    "--terra-menu-fullscreen-header-buttons-padding": "4px",
+    "--terra-menu-background-color": "#fff",
+    "--terra-menu-foreground-color": "#1c1f21",
+    "--terra-menu-submenu-foreground-color": "#1c1f21"
+  },
+  "MenuItem": {
+    "--terra-menu-item-hover-color": "#ebf6fd",
+    "--terra-menu-item-text-color": "#1c1f21",
+    "--terra-menu-item-text-font-size": "13px",
+    "--terra-menu-item-checkmark-icon-color": "#1c1f21",
+    "--terra-menu-item-checkmark-icon-font-size": "12px",
+    "--terra-menu-item-chevron-icon-color": "#9b9fa1",
+    "--terra-menu-item-chevron-icon-font-size": "10px"
+  },
+  "MockThemeComponent": {
+    "--terra-mock-background-color": "#000",
+    "--terra-mock-color": "#fff",
+    "--terra-mock-display": "inline"
+  },
+  "Popup": {
+    "--terra-popup-element-z-index": "7001"
+  },
+  "PopupArrow": {
+    "--terra-popup-arrow-size": "11px",
+    "--terra-popup-arrow-z-index": "7002",
+    "--terra-popup-arrow-inner-size": "10px",
+    "--terra-popup-arrow-inner-z-index": "7003",
+    "--terra-popup-arrow-color": "#b2b5b6",
+    "--terra-popup-arrow-inner-inset-size": "2px",
+    "--terra-popup-arrow-inner-color": "#fff"
+  },
+  "PopupContent": {
+    "--terra-popup-content-header-border-bottom": "1px solid #dedfe0",
+    "--terra-popup-content-close-background": "0 0",
+    "--terra-popup-content-close-border": "0",
+    "--terra-popup-content-close-color": "#111",
+    "--terra-popup-content-close-font-size": "1.45rem",
+    "--terra-popup-content-close-font-weight": "bold",
+    "--terra-popup-content-close-line-height": "0",
+    "--terra-popup-content-close-margin-left": "0.3571rem",
+    "--terra-popup-content-close-margin-right": "0",
+    "--terra-popup-content-close-padding": "4px",
+    "--terra-popup-content-close-hover-color": "#999",
+    "--terra-popup-content-inner-background-color": "#fff",
+    "--terra-popup-content-inner-border": "1px solid #b2b5b6",
+    "--terra-popup-content-inner-box-shadow, 0 4px 8px 0 rgba(0, 0, 0, 0.35), 0 4px 3px -3px rgba(0, 0, 0": "0.5)",
```

Comment:
Logged an issue about this malformatting: https://github.com/cerner/terra-core/issues/818
It should not impact users much as I don't think anyone is using this file other than us to generate the list of themable components here on https://terra-ui.herokuapp.com/getting-started#theming. For that we only use the top level keys.